app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -128,21 +128,34 @@ import org.oppia.android.util.networking.NetworkConnectionUtilDebugModule
 import org.oppia.android.util.parser.html.HtmlParserEntityTypeModule
 import org.oppia.android.util.parser.image.GlideImageLoaderModule
 import org.oppia.android.util.parser.image.ImageParsingModule
-import org.oppia.android.util.platformparameter.EnableExtraTopicTabsUi
+import org.oppia.android.util.platformparameter.EnableTopicInfoTab
+import org.oppia.android.util.platformparameter.EnableTopicPracticeTab
 import org.oppia.android.util.platformparameter.PlatformParameterValue
 import org.oppia.android.util.profile.CurrentUserProfileIdIntentDecorator.decorateWithUserProfileId
 import org.robolectric.annotation.Config
 import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
+// When both extra tabs (info and practice) are enabled
 private const val INFO_TAB_POSITION = 0
 private const val LESSON_TAB_POSITION = 1
 private const val PRACTICE_TAB_POSITION = 2
 private const val REVISION_TAB_POSITION = 3
+
+// When both extra tabs (info and practice) are disabled
 private const val LESSON_TAB_POSITION_EXTRA_TABS_DISABLED = 0
 private const val REVISION_TAB_POSITION_EXTRA_TABS_DISABLED = 1
 
+// When only the info tab is enabled
+private const val LESSON_TAB_POSITION_INFO_ENABLED_PRACTICE_DISABLED = 1
+private const val REVISION_TAB_POSITION_INFO_ENABLED_PRACTICE_DISABLED = 2
+
+// When only the practice tab is enabled
```

Comment:
Apply to others also.

```suggestion
// When only the practice tab is enabled.
```

Reply:
Done.